Tribler/Test/Community/Multichain/StatisticsDispaly/test_statistics_database.py

```diff
@@ -0,0 +1,73 @@
+from Tribler.Test.test_as_server import BaseTestCase
```

Comment:
Add module description at the top of module. Look at [PEP 257](https://www.python.org/dev/peps/pep-0257/) for more information.